spec/models/requests/requestable_spec.rb

```diff
@@ -795,36 +786,7 @@
 
     describe '#location_label' do
       it 'has a location label' do
-        expect(requestable.location_label).to eq('Firestone Library - Stacks')
-      end
-    end
-
-    describe "#held_at_marquand_library?" do
-      it "is not marquand" do
-        expect(requestable).not_to be_held_at_marquand_library
-      end
-    end
-
-    describe "#available?" do
-      it "is not available" do
-        expect(requestable).not_to be_available
-      end
-    end
-  end
-
-  context 'Pending Order materials' do
```

Comment:
Did the concept of pending order get dropped?

Reply:
Yes. That was a voyager construct.